app/controllers/events_controller.rb

```diff
@@ -1,6 +1,11 @@
 class EventsController < ApplicationController
   def index
     @events = meetup.self_events
+    @events = @events.select do |event|
```

Comment:
@Vitormdias Great! I'm just a little worried about the method's name, what do you think about `hosted_by_user?` or something like that?